ha/util/message_bus.py

```diff
@@ -144,6 +146,16 @@ def stop(self):
 
 class MessageBus:
     ADMIN_ID = "ha_admin"
+    _DELIM = ">"
+    HA_GLOBAL_INDEX = "ha_conf"
```

Comment:
Both _DELIM and HA_GLOBAL_INDEX should be imported from const.py file in ha

Reply:
As it is utils code keeping it separate from ha code, so in future if we need to move this code to utils it would be easy.
Already there are some const present in same file which are kept it here with same objective.

Reply:
IMHO, HA_GLOBAL_INDEX is specific to ha(for ha.conf file). So, we can not move it to utils. And _DELIM must have been defined in utils constants. So, I think we don't need to define it here separately.

Reply:
Done. Used values from const.py